db/migrate/20220308125926_create_projects.rb

```diff
@@ -0,0 +1,12 @@
+class CreateProjects < ActiveRecord::Migration[7.0]
+  def change
+    create_table :projects do |t|
+      t.string :title
+      t.string :discription
```

Comment:
Do you have any specific problem with spelling of `discription` ???

Reply:
fixed